modules/ROOT/pages/api-experience-hub-lightning-components.adoc

```diff
@@ -1,57 +1,101 @@
+
+To delete a component, click the delete icon and click *Delete* from the dialog.
+
+== API Console
 
-* API Console
-+
 The interactive API Console component shows information about your APIs and their methods. Endpoints are displayed in alphabetical order.
 +
 Console components include API Console, API Console Documentation, API Console Instances, API Console Specification, and API Console Request Panel. In the Builder, you can add multiple console components to the same page. Changes to a console component are visible in all other console components on that page that display the same API. For example, selecting the Summary tab in an API Console results in Summary tab selection in all API Consoles on the same page that display the same API.
 +
 The component shows the public API instances and documentation from Anypoint Exchange.
 
-* Asset Detail Header
-+
+== Asset Detail Header
+
 The Asset Details Header component displays the API header, including icon, name, contact email, and description.
-+
+
 The API icon, name, and description shown are set in the API Experience Hub control panel when you add an API version to your community. The contact email is set in Anypoint Exchange.
-+
-You can specify values for the following parameters of the API Header:
 
-* Asset Version 
-+
+//You can enter or select the following Asset API Header properties:
+//
+//* Record id
+//* Show API Contact
+//* Show API Clients
+//* Show instance URL
+//* Show categories
+//* Show tags
+
+== Asset Version 
+
 Shows the API version from Exchange when only one version is visible.
-* Version State
-+
+
+== Version State
+
 Shows the API version lifecycle state.
 * Asset Documentation Viewer
-+
+
 The API Documentation Viewer component shows an API’s documentation from Anypoint Exchange. The documentation can have multiple pages, and each page can have multiple sections.
 
-* Assets Carousel
+== Assets Carousel
+
+The API Carousel component shows a horizontally scrollable set of API cards that displays on the Home page. Each card shows an API.
+
+You can specify values for the following Assets Carousel properties:
+
+* Title
 +
-The API Carousel component shows a horizontally scrollable set of API cards. Each card shows an API.
+Title of the carousel.
+* Subtitle
++
+A subtitle that displays after the title of the carousel.
 
-* Assets Catalog
+* Cards to display
 +
+The number of cards to display in the carousel. The carousel can display a minimum of three cards and a maximum of fifteen cards.
+
+=== Edit the Titles in the API Carousel
+You can edit the title and subtitle that appears for the API Carousel.
+
+. From the Manage your API portal pages, click *Preview and publish your portal*.
+. In the *Home* page, click the *Asset Carousel* area to edit the Assets Carousel component.
+. In the *Title* field, enter a name.
+. In the *Subtitle* fieldI, enter a name.
```

Comment:
When the user changes the number of cards, and then changes the carousel subtitle, this key {numberOfAssets} on the subtitle will replaced with the number of Cards to display the set:
Description added for this field: `Modify subtitle of carousel, '{numberOfAssets}' will be replaced with the numbers of assets displayed`

@JennyHajee Eval if this need to add this

Reply:
Thanks, I will add the steps to add the card count to the subtitle